docs/main/user-guides/local-storage-user-guide/local-pv-zfs/advanced-operations/zfs-backup-restore.md

```diff
@@ -47,13 +47,13 @@ aws_secret_access_key = minio123
 Install Velero using the command below:
 
 ```
-velero install --provider aws --bucket velero --secret-file /home/pawan/velero/credentials-minio --plugins velero/velero-plugin-for-aws:v1.0.0-beta.1 --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000 --use-volume-snapshots=true --use-restic
+velero install --provider aws --bucket velero --secret-file /home/pawan/velero/credentials-minio --plugins velero/velero-plugin-for-aws:v1.0.0-beta.1 --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000 --use-volume-snapshots=true --use-node-agent --uploader-type restic
```

Comment:
Are these the changes needed to make this work with the latest Velero?

If we use the restic way, does it work for block mode of volumes as well?

Reply:
yes , tested it against the latest velero. `--use-restic` flag was deprecated long ago and installation fails if we use `--use-restic` flag.

Reply:
What about the block mode of volumes?

Reply:
yes, tested it against the block volumes as well using this docs https://github.com/openebs/zfs-localpv/blob/develop/docs/raw-block-volume.md